### PR TITLE
Remove unnecessary builtins imports.

### PR DESCRIPTION
**Pull request by oliverchang**
Also fix some minor remaining lint issues for the affected files.

### CODE REVIEW

File: docker/oss-fuzz/host/start_host.py [hunk 1]
```diff
@@ -14,9 +14,6 @@
 """Start host."""
 from __future__ import print_function
```
Comment (inferno-chromium) [inline docker/oss-fuzz/host/start_host.py:15]: Should all these from __future__ go away too.

**Review by inferno-chromium — APPROVED** [review]
Amazing!

File: docker/oss-fuzz/host/start_host.py (hunk 1 above)
Comment (oliverchang) [reply docker/oss-fuzz/host/start_host.py:15]: Will do this in another PR :)

**Pull request merged**